pyrevolve/revolve_bot/revolve_module.py

```diff
@@ -265,6 +323,38 @@ def to_sdf(self, tree_depth='', parent_link=None, child_link=None):
         parent_link.append(imu_sensor)
         return visual, collision, imu_sensor
 
+    def count_module_connections(self) -> int:
+        """
+        Count how many connections the module has.
+        Connected TouchSensor and BrickSensor Modules are ignored
+        :return: number of connections
+        """
+        # The CoreModule does not have a parent module
+        return super(CoreModule, self).count_module_connections() - 1
+
+    def is_folding(self) -> bool:
```

Comment:
Why does is_folding need to be overwritten by CoreModule?

Reply:
Because in the other function I assume you always have one connection already, which is the connection to the parent. In this case the assumption is broken.

Reply:
Is there anything we can reuse?